sympy/solvers/tests/test_ode.py

```diff
@@ -3003,10 +2987,16 @@ def test_2nd_power_series_ordinary():
         -x**4/24 + x**3/6 + 1) + C1*x*(x**3/24 + x**2/6 - x/2
         + 1) + O(x**6))
 
-    eq = f(x).diff(x, 2) + x*f(x)
-    assert classify_ode(eq) == ('2nd_power_series_ordinary',)
-    assert dsolve(eq, n=7) == Eq(f(x), C2*(
-        x**6/180 - x**3/6 + 1) + C1*x*(-x**3/12 + 1) + O(x**7))
+def test_Airy_equation():
+    C1, C2 = symbols("C1 C2")
+    from sympy.functions import airyai, airybi
+    eq = f(x).diff(x, 2) - x*f(x)
+    assert classify_ode(eq) == ("2nd_linear_airy",'2nd_power_series_ordinary')
+    assert dsolve(eq) == Eq(f(x), C1*airyai(x) + C2*airybi(x))
+    eq = f(x).diff(x, 2) + 2*x*f(x)
+    assert classify_ode(eq) == ("2nd_linear_airy",'2nd_power_series_ordinary')
+    assert str(dsolve(eq)) == "Eq(f(x), C1*airyai(-2**(1/3)*x) + C2*airybi(-2**(1/3)*x))"
```

Comment:
Don't use str for comparing solutions. And can you add checkodesol?

Reply:
okkk